include/re_sipsess.h

```diff
@@ -74,3 +83,4 @@ void sipsess_close_all(struct sipsess_sock *sock);
 struct sip_dialog *sipsess_dialog(const struct sipsess *sess);
 void sipsess_abort(struct sipsess *sess);
 bool sipsess_ack_pending(const struct sipsess *sess);
+enum sdp_neg_state sipsess_sdp_neg_state(const struct sipsess *sess);
```

Comment:
Does the state have to be exposed to the application?

Reply:
I believe so. E.g. the application can check if the state is `SDP_NEG_PREVIEW_ANSWER` (unreliable SDP answer was sent in 1XX response) and in this case it MUST send the exact same SDP the 200 OK. Ideally, re would not allow non RFC conformant usage, but we're not there yet. That would take more refactoring that could be done i a subsequent step.

Reply:
The following BareSIP PR makes use of the state:

- https://github.com/baresip/baresip/pull/2818